### DISCUSSION TITLE
Map-match pedestrian movement on cycleway

### DISCUSSION THREAD

**Discussion by xburir** [valhalla/valhalla#4682] [Q&A] [answered]
I am mapmatching a route, but the map-match stops on a cycleway, I was wondering, if there is way to set the options so it would mapmatch pedestrian movement on a cycleway too.

**Comment by kevinkreiser** [comment, marked as answer]
yes in fact there is. we have costing options that allow you to tell the router or map matcher that you dont care what "access" a given way has. this option is called "ignore_acces".  you can see more here: https://valhalla.github.io/valhalla/api/turn-by-turn/api-reference/#costing-options its under the automobile options section but it applies to all costings.

**Reply by xburir** [reply]
I thought this costing option could be used for `auto` `bus` and `truck` only since it is written above the options table, anyway I used it and it seems it is working, thanks
